packages/plugins/components/package.json

```diff
@@ -13,8 +13,8 @@
     },
     "scripts": {
         "build": "yarn build:esm && yarn build:cjs",
-        "build:esm": "tsc --module esnext --outDir lib/esm",
-        "build:cjs": "tsc",
+        "build:esm": "tsc --module esnext --outDir lib/esm -p ./tsconfig.build.json",
+        "build:cjs": "tsc -p ./tsconfig.build.json",
```

Comment:
nit: prefer use --project. easier to understand 

 --project, -p  Compile the project given the path to its configuration file, or to a folder with a 'tsconfig.json'.

Reply:
sounds good, will update